js/libraries/jbox/jBox.css

```diff
@@ -19,7 +19,6 @@
 .jBox-container {
 	background: #fff;
 	border:1px solid #ffbb00;
-	max-width:180px;
```

Comment:
Not sure if it's a good idea to patch this inside the library, as this will most likely be forgotten and overwritten when we update the library next time. Can we set this somewhere else?